buildSrc/src/main/java/org/elasticsearch/gradle/precommit/DependencyLicensesTask.java

```diff
@@ -210,8 +210,7 @@ private void failIfAnyMissing(String item, Boolean exists, String type) {
     }
 
     private void checkDependencies(Map<String, Boolean> licenses, Map<String, Boolean> notices, Set<File> shaFiles)
-        throws NoSuchAlgorithmException,
-        IOException {
+        throws NoSuchAlgorithmException, IOException {
```

Comment:
Just a question: I wonder if we want to give it another indent here for the `throws` if the `)` is not on the same line as the `throws` so it's one deeper than the method body? Is that possible?

Reply:
I had another fiddle with the settings but this doesn't seem to be possible.